scripts/linters/css_linter.py

```diff
@@ -67,10 +67,7 @@ def _get_trimmed_error_output(css_lint_output):
             # "x"(\u2716) and a message-id in the end. We are capturing these
             # and then replacing them with empty string('').
             if re.search(r'^\d+:\d+', line.lstrip()):
-                # Replacing message-id with an empty string('').
-                line = re.sub(r'(\w+-*)+$', '', line)
```

Comment:
This re.sub is taking an infinite time to complete because of unknown reasons.

```
import re

line = u' 298:3  \u2716  Unexpected duplicate selector ".conversation-skin-supplemental-card-container", first used at line 112   no-duplicate-selectors'

re.sub(r'(\w+-*)+$', '', line) // This line never completes.
```

Also, I found that we should show the message-id to the users so that they can do further investigation for the lint issue if required.

Reply:
FYI, sometimes regexes run into issues of infinite recursion -- see https://stackoverflow.com/questions/8010005/python-re-infinite-execution. Not quite sure if that's the case here but just wanted to note it FYI.

I agree that using .replace() makes sense.